shared/src/main/scala/Dictionary.scala

```diff
@@ -0,0 +1,10 @@
+package vyxal
+object Dictionary:
+  var shortDictionary: Seq[String] = Seq()
+  var longDictionary: Seq[String] = Seq()
+  var initalised: Boolean = false
+
+  def initalise(short: Seq[String], long: Seq[String]): Unit =
```

Comment:
This should be `initialize`. Yes, with a "z." It is my right as an American citizen to not have to use your British "English." I will use "z" and put my periods inside my quotes and you can't stop me.

Reply:
Skill issue + Australian english + resolved conversation + not changed

Reply:
I was joking about the "z," but you really do need to change the spelling